### DISCUSSION TITLE
Can I rearrange the colors?

### DISCUSSION THREAD

**Discussion by lifehaschanged** [RyotaUshio/obsidian-pdf-plus#143] [Q&A] [answered]
Is there a way to rearrange colors?
I tried with renaming them but they got mixed up if the name of the color is already taken by the other color. > If I right click in the PDF book tab, the colors are not the same as in the PDF++ options.

**Comment by RyotaUshio** [comment, marked as answer]
Sorry this plugin doesn't offer the GUI for rearranging colors. I will consider adding it in the future.

But I guess the following workaround will work:

1. Disable PDF++.
2. Open `.obsidian/plugins/pdf-plus/data.json` in some text editor.
3. Find an item called "colors". It will look like this:

```
"colors": {
    "Yellow": "#ffd000",
    "Red": "#ea5252",
    "Note": "#086ddd",
    "Important": "#bb61e5"
  },
```

4. Rearrange these key-value pairs as you like.
5. Save the change and re-enable PDF++.

**Reply by codingThome** [reply]
You could do this in the Plugin-Settings-Menu - there you can delete and add colors - so you can set your own order

**Comment by lifehaschanged** [comment]
> Sorry this plugin doesn't offer the GUI for rearranging colors. I will consider adding it in the future.
> 
> But I guess the following workaround will work:
> ...

Thanks, this workaround works. But sure a GUI would be fine :)


> You could do this in the Plugin-Settings-Menu - there you can delete and add colors - so you can set your own order

Yes, thank you too. This is surely ok if you have not many colors :)
